src/test/java/com/example/bank/transfer/TransferServiceIT.java

```diff
@@ -1,106 +0,0 @@
-package com.example.bank.transfer;
-
-import com.example.bank.account.AccountRepository;
-import com.example.bank.account.model.Account;
-import com.example.bank.context.TransferOwnContext;
-import com.example.bank.exception.ExceptionType;
-import com.example.bank.exception.RestException;
-import com.example.bank.transfer.model.TransferHistory;
-import com.example.bank.transfer.model.TransferType;
-import jakarta.validation.ConstraintViolationException;
-import org.assertj.core.api.SoftAssertions;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
-import org.springframework.test.annotation.DirtiesContext;
-import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.ContextConfiguration;
-import org.springframework.test.context.TestPropertySource;
-import org.springframework.test.context.jdbc.Sql;
-import org.springframework.test.context.junit.jupiter.SpringExtension;
-import org.springframework.transaction.annotation.Transactional;
-
-import java.math.BigDecimal;
-import java.math.RoundingMode;
-import java.security.Principal;
-
-@ExtendWith(SpringExtension.class)
-@ContextConfiguration(classes = TransferOwnContext.class)
-@ActiveProfiles("test")
-@Sql(scripts = {"classpath:schema.sql", "classpath:data.sql"}, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_CLASS)
-@TestPropertySource("classpath:application-test.yml")
-@Transactional
-@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
-class TransferServiceIT {
```

Comment:
shouldProcessBankTransferCorrectly() – Poprawienie porównania kwot
Potrzebujesz tego DirtiesContext? 
- Brak testu dla nieistniejącego konta
-  shouldThrowRestExceptionWhenBalanceIsInsufficient() – Można poprawić dodajac weryfikacje przed i po 
-  Brak testu dla przelewu na to samo konto
- Brak testu dla negatywnej kwoty przelewu
- Brak testu dla pustego tytułu przelewu
-